src/components/Local.vue

```diff
@@ -73,19 +83,20 @@ export default {
 
   data: function() {
     return {
-      stringCopied: "Copied to clipboard",
+      stringCopied: "Copied to clipboard.",
       newClip: "",
       clips: [],
-      preview: "Here you'll see preview as soon as you start typing.",
+      preview: "Start typing to see preview",
       emptyString: false,
       copied: false,
       notCopied: false,
-      emptyStringText: "Cannot add empty string",
-      timeout: 3000,
+      emptyStringText: "Cannot add empty clip.",
```

Comment:
what is this timeout for ?

Reply:
It's for the snackbar notifications to disappear, the tiny ones that gets shown when user tries to add an empty clip